### PR TITLE
Implement a window resize/draggable mode

### LINKED ISSUE #71: can't copy text from chat with mouse
If I try to highlight text from the chat pane in order to copy and paste, it just drags the pane around.

### PR DESCRIPTION
**Pull request by ChrisBr**
This will fix #71 because it was not possible to c&p from the chat while
resizing/dragging was enabled.

### CODE REVIEW

File: src/app/components/videochat/jh-video-chat.directive.js [hunk 1]
```diff
@@ -192,6 +192,27 @@
         hotkeys.toggleCheatSheet();
       }
 
+      function enableWindowResizeMode() {
+        var vm = this;
+        vm.windowResizeModeOn = !vm.windowResizeModeOn;
+        vm.gridsterOpts = {
+          resizable: {
+            enabled: vm.windowResizeModeOn,
+            stop: function (event, $element) {
```
Comment (imobach) [inline src/app/components/videochat/jh-video-chat.directive.js]: I think that if you move the `stop` definition to the original `gridsterOpts` building, this function could be something like:

```js
function enableWindowResizeMode() {
  var vm = this;
  vm.windowResizeModeOn = !vm.windowResizeModeOn;
  vm.gridsterOpts.resizable.enabled = vm.windowResizeModeOn;
  vm.gridsterOpts.draggable.enabled = vm.windowResizeModeOn;
}
```

What do you think?

Comment (ChrisBr) [inline src/app/components/videochat/jh-video-chat.directive.js]: Ahh, nice! Works :)

**Comment by ChrisBr** [comment]
@imobach @ancorgs updated!

File: src/app/components/videochat/jh-video-chat.directive.js [hunk 2]
```diff
@@ -192,6 +200,13 @@
         hotkeys.toggleCheatSheet();
       }
 
+      function enableWindowResizeMode() {
```
Comment (ancorgs) [inline src/app/components/videochat/jh-video-chat.directive.js]: better use `toggleWindowResizeMode` please

**Comment by ancorgs** [comment]
LGTM!

**Pull request merged**